### ISSUE TITLE
GUI Text Change Request: Unlock Wallet Pop-Up on Spend

### ISSUE THREAD

**Issue by Rock-N-Troll** [Veil-Project/veil#532]
![image](https://user-images.githubusercontent.com/34344520/56537047-8e7cfb80-652d-11e9-9e34-fc2ef4d0633a.png)

"Save" is not the right word for unlocking wallet or unlocking wallet for spending.

**Comment by veilgets** [comment]
Bounty: 83.3

**Comment by MatWaller** [comment]
Oh a bounty! 

Pull Request Outstanding :)

**Comment by veilgets** [comment]
Developer: MatWaller

**Comment by MatWaller** [comment]
Bounty Address
```
sv1qqpdsfcqvfcljtcq8txc8kc2dw4e9upj69wnnfv7j0ufcsrv7trs9lqpqfscke6jj92aj4aup387ggk7mathznr8hz3zde6ut87jdwphrns7qqqq00jn8w
```

**Comment by MatWaller** [comment]
Closed with #549
